lib/errors.js

```diff
@@ -86,12 +86,14 @@ util.inherits(errors.ResourceNotFoundError, errors.SmartcarError);
  *
  * @extends SmartcarError
  * @param message - an error description to set
+ * @param code - a vehicle state error code
```

Comment:
thoughts on putting a link to the section of API reference that documents the codes for vehicle state error?

Reply:
I like the idea, done!